stubs/setuptools/setuptools/command/easy_install.pyi

```diff
@@ -127,17 +127,17 @@ class ScriptWriter:
     template: Any
     command_spec_class: Any
     @classmethod
-    def get_script_args(cls, dist, executable: Any | None = ..., wininst: bool = ...): ...
+    def get_script_args(cls, dist, executable: Any | None = ..., wininst: bool = ...) -> Iterable[tuple[str, str]]: ...
```

Comment:
`Iterable` is correct, but `Iterator` or `Generator` might better, as that is more specific. Also applies to `get_args`.